stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -81,6 +86,10 @@ spec:
               touch "${RABBITMQ_MNESIA_DIR}/force_load"
             fi
             {{- end }}
+            {{- if and (.Values.securityContext.runAsUser) (.Values.securityContext.fsGroup) }}
+            chown {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /var/lib/rabbitmq/.erlang.cookie
+            chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /etc/rabbitmq
```

Comment:
Why are you chowning the configmap dir? I don't think that is the right thing to do and indeed may fail in some environments (or be ignored). Most likely these two lines should be combined and just `chown -R .. /var/lib/rabbitmq`. The linked PR also just chowned the data directory.

Reply:
Since we've changed the init container to now run as root, the first thing it does is copy over the configmap to /etc/rabbitmq ([in this line](https://github.com/helm/charts/pull/21344/files/cc39bc7e431075b7b8d3a4760c8689e058cfa278#diff-e00e6fcd2c8b56c1bd9161e193b126fdR82)). I just want to be 100% sure that anything it copies over is owned by the correct user.

Reply:
Ok, fine to keep the /etc/rabbitmq chown, but still think you should `chown -R .. /var/lib/rabbitmq` like the related PR did. For example, you only did the cookie but already missed the force_load file. Might as well do all.

Reply:
@steven-sheehy  - you're right. Change made